CMakeLists.txt

```diff
@@ -436,7 +435,7 @@ find_package(OpenMP)
 find_library(LIBSNDFILE_LIBRARY sndfile)
 
 if(NOT LIBSNDFILE_LIBRARY AND WIN32)
-find_library(LIBSNDFILE_LIBRARY sndfile-1)
+find_library(LIBSNDFILE_LIBRARY libsndfile-1)
```

Comment:
This could be `find_library(LIBSNDFILE_LIBRARY sndfile libsndfile-1)` above instead of this if?